bpf/command.bpf.c

```diff
@@ -0,0 +1,146 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>       /* most used helpers: SEC, __always_inline, etc */
+#include <bpf/bpf_core_read.h>     /* for BPF CO-RE helpers */
+#include <bpf/bpf_tracing.h>       /* for getting kprobe arguments */
+
+#include "helpers.h"
+
+#define ARGSIZE  128
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
```

Comment:
Can this be Apache License Version 2?

Reply:
I think it has the same license requirements as a kernel module: https://elixir.bootlin.com/linux/v5.8/source/include/linux/module.h#L182

Reply:
@klizhentas Is `Dual BSD/GPL` okay?

Reply:
Can you tell me more about how we distribute it? Are we using dual licensed components linking from here and deriving license of this file from those? How are we going to distribute the program?